src/swaggers/openpix.yml

```diff
@@ -243,41 +248,56 @@ paths:
             return_existing:
               value: true
       description: Endpoint to create a new Charge for a customer
+      x-codeSamples:
```

Comment:
![image](https://user-images.githubusercontent.com/2005841/193677807-5120c0c1-9288-4eca-900b-dbb4cae72ed4.png)

can we autogenerate this?